backend/src/api/cycle.js

```diff
@@ -36,28 +50,25 @@ router.get(
   })
 )
 
-// get all cycles belonging to a workspace (either current, outdated, or both)
-// TODO: when authentication server is integrated, only show the cycles
-// to the workspace's owner. Otherwise, 403.
+// get all cycles belonging to the workspace that a user belongs to (either current, outdated, or both)
 router.get(
-  '/workspace/:workspaceName',
+  '/workspace',
```

Comment:
note to self: the way we get workspaces in the frontend is going to have to be changed to getting ids from user

Reply:
I changed the rendering issues on the frontend already